lib/glimesh_web/templates/user_payments/setup.html.eex

```diff
@@ -0,0 +1,45 @@
+<div class="container">
+    <h2 class="mt-4"><%= gettext("Setup Glimesh Payouts") %></h2>
+
+    <%= if @can_payments do %>
```

Comment:
I'm not gettexting a lot of this page yet because it is nothing near the final text.